config/webpack/serverWebpackConfig.js

```diff
@@ -103,14 +105,14 @@ const configureServer = () => {
   // eval works well for the SSR bundle because it's the fastest and shows
   // lines in the server bundle which is good for debugging SSR
   // The default of cheap-module-source-map is slow and provides poor info.
-  serverWebpackConfig.devtool = 'eval'
+  serverWebpackConfig.devtool = 'eval';
 
   // If using the default 'web', then libraries like Emotion and loadable-components
   // break with SSR. The fix is to use a node renderer and change the target.
   // If using the React on Rails Pro node server renderer, uncomment the next line
   // serverWebpackConfig.target = 'node'
 
-  return serverWebpackConfig
-}
+  return serverWebpackConfig;
+};
 
-module.exports = configureServer
+module.exports = configureServer;
```

Comment:
Why all the formatting changes? Are we adding settings for eslint and rubocop?

Reply:
Reverted